lib/versioner/external.go

```diff
@@ -51,16 +50,11 @@ func (v External) Archive(filePath string) error {
 
 	l.Debugln("archiving", filePath)
 
-	inFolderPath, err := filepath.Rel(v.folderPath, filePath)
-	if err != nil {
-		return err
-	}
-
 	if v.command == "" {
 		return errors.New("Versioner: command is empty, please enter a valid command")
 	}
 
-	cmd := exec.Command(v.command, v.folderPath, inFolderPath)
+	cmd := exec.Command(v.command, v.filesystem.Type(), v.filesystem.URI(), filePath)
```

Comment:
Compatibility: if the filesystem type is `basic`, call it with the old parameter format?